src/app/tests/suites/certification/Test_TC_OPCREDS_1_2.yaml

```
@@ -23,87 +23,68 @@ config:
-          [1656324429.813513][17795:17800] CHIP:TOO:     [8]: 65529
-          [1656324429.813575][17795:17800] CHIP:TOO:     [9]: 65531
-          [1656324429.813636][17795:17800] CHIP:TOO:     [10]: 65532
-          [1656324429.813697][17795:17800] CHIP:TOO:     [11]: 65533
-      disabled: true
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains:
+                  [
+                      0xFFFD,
+                      0xFFFC,
+                      0xFFFB,
+                      0xFFF9,
+                      0xFFF8,
+                      0x00,
+                      0x01,
+                      0x02,
+                      0x03,
+                      0x04,
+                      0x05,
+                  ]
 
     - label: "TH reads EventList from DUT"
       verification: |
           out of scope for V1.0
       disabled: true
 
     - label: "TH reads AcceptedCommandList from DUT"
-      verification: |
-          ./chip-tool operationalcredentials read  accepted-command-list 1 0
-
-          Verify on the TH Log:
-
-          [1657021609.426857][7131:7136] CHIP:TOO: Endpoint: 0 Cluster: 0x0000_003E Attribute 0x0000_FFF9 DataVersion: 364716653
-          [1657021609.426944][7131:7136] CHIP:TOO:   AcceptedCommandList: 8 entries
-          [1657021609.426978][7131:7136] CHIP:TOO:     [1]: 0
-          [1657021609.427005][7131:7136] CHIP:TOO:     [2]: 2
-          [1657021609.427032][7131:7136] CHIP:TOO:     [3]: 4
-          [1657021609.427059][7131:7136] CHIP:TOO:     [4]: 6
-          [1657021609.427085][7131:7136] CHIP:TOO:     [5]: 7
-          [1657021609.427111][7131:7136] CHIP:TOO:     [6]: 9
-          [1657021609.427138][7131:7136] CHIP:TOO:     [7]: 10
-          [1657021609.427165][7131:7136] CHIP:TOO:     [8]: 11
-      disabled: true
+      command: "readAttribute"
+      attribute: "AcceptedCommandList"
+      response:
+          constraints:
+              type: list
+              contains: [0x00, 0x02, 0x04, 0x06, 0x07, 0x09, 0x0a, 0x0b]
 
     - label: "TH reads GeneratedCommandList from DUT"
-      verification: |
-          ./chip-tool operationalcredentials read generated-command-list 1 0
-
-          Verify on the TH Log:
-
-          [1651215270.028855][3260:3265] CHIP:TOO: Endpoint: 0 Cluster: 0x0000_003E Attribute 0x0000_FFF8 DataVersion: 3662641304
-          [1651215270.028955][3260:3265] CHIP:TOO:   GeneratedCommandList: 4 entries
-          [1651215270.029001][3260:3265] CHIP:TOO:     [1]: 1
-          [1651215270.029039][3260:3265] CHIP:TOO:     [2]: 3
-          [1651215270.029076][3260:3265] CHIP:TOO:     [3]: 5
-          [1651215270.029112][3260:3265] CHIP:TOO:     [4]: 8
-      disabled: true
+      command: "readAttribute"
+      attribute: "GeneratedCommandList"
+      response:
+          constraints:
+              type: list
+              contains: [0x01, 0x02, 0x03, 0x05, 0x08]
```

```suggestion
              contains: [0x01, 0x03, 0x05, 0x08]
```
and needs regen.

Also needs regen of darwin-framework-tool, right?

---

Doesn't look like it, I just ran regen, nothing.

---

So there are a few things:

1. The test is not actually getting automated because it's still in the "manual tests" list in `src/app/tests/suites/tests.js`
2. As you noted, after merging to tip the incorrect constraints are failing.

---

oh lord...why were those changed? Just for fun?

---

The weren't change.  They were just no-ops before and now we try to enforce them.

As in, the constraint lines should say "int16u" (which is what data model calls the type), not "uint16".

---

makes sense - changed now.